src/main/kotlin/app/revanced/patcher/resource/Resource.kt

```diff
@@ -0,0 +1,167 @@
+package app.revanced.patcher.resource
```

Comment:
This file does not belong in ReVanced Patcher because it is strictly dealing with resource files from arsclib. Please create a separate module that wraps around `arsclib` and use it in ReVanced Patcher.